src/components/Navbar/Logo/Logo.tsx

```diff
@@ -7,5 +8,5 @@ type LogoProps = {
 export const Logo = ({ variant }: LogoProps) => {
   const logo = variant === 'default' ? <JumperLogo /> : <JumperLearnLogo />;
 
-  return logo;
+  return <LogoWrapper>{logo}</LogoWrapper>;
```

Comment:
why this wrapper is needed?

Reply:
it is kinda this, the 404 page can only be server side from what I found